tests/torch/experimental/search_building_blocks/test_search_building_blocks.py

```diff
@@ -0,0 +1,97 @@
+    ['Inception3/InceptionC[Mixed_6c]/BasicConv2d[branch7x7dbl_2]/relu__0',
+     'Inception3/InceptionC[Mixed_6c]/BasicConv2d[branch7x7dbl_4]/relu__0'],
+    ['Inception3/InceptionC[Mixed_6b]/cat_0', 'Inception3/InceptionC[Mixed_6c]/cat_0'],
+    ['Inception3/InceptionC[Mixed_6d]/BasicConv2d[branch7x7dbl_2]/relu__0',
+     'Inception3/InceptionC[Mixed_6d]/BasicConv2d[branch7x7dbl_4]/relu__0'],
+    ['Inception3/InceptionC[Mixed_6c]/cat_0', 'Inception3/InceptionC[Mixed_6d]/cat_0'],
+    ['Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_2]/relu__0',
+     'Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_4]/relu__0'],
+    ['Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_3]/relu__0',
+     'Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_5]/relu__0'],
+    ['Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_2]/relu__0',
+     'Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch7x7dbl_5]/relu__0'],
+    ['Inception3/InceptionC[Mixed_6d]/cat_0', 'Inception3/InceptionC[Mixed_6e]/cat_0'],
+    ['Inception3/InceptionE[Mixed_7b]/cat_2', 'Inception3/InceptionE[Mixed_7c]/cat_2']]
+
+REF_BUILDING_BLOCKS_FOR_SQUEEZENET1_0 = [
+    ['SqueezeNet/Sequential[features]/Fire[3]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[4]/ReLU[squeeze_activation]/relu__0'],
+    ['SqueezeNet/Sequential[features]/Fire[4]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[4]/cat_0'],
+    ['SqueezeNet/Sequential[features]/Fire[7]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[7]/cat_0'],
+    ['SqueezeNet/Sequential[features]/Fire[8]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[9]/ReLU[squeeze_activation]/relu__0'],
+    ['SqueezeNet/Sequential[features]/Fire[9]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[9]/cat_0'],
+    ['SqueezeNet/Sequential[features]/Fire[12]/ReLU[squeeze_activation]/relu__0',
+     'SqueezeNet/Sequential[features]/Fire[12]/cat_0']]
+
+@pytest.mark.parametrize('model_creator, input_sizes, ref_building_blocks',
+                         ((ResNet50, RESNET50_INPUT_SIZE, REF_BUILDING_BLOCKS_FOR_RESNET),
+                          (MobileNetV2, RESNET50_INPUT_SIZE, REF_BUILDING_BLOCKS_FOR_MOBILENETV2),
+                          (Inception3, INCEPTION_INPUT_SIZE, REF_BUILDING_BLOCKS_FOR_INCEPTIONV3),
+                          (squeezenet1_0, RESNET50_INPUT_SIZE, REF_BUILDING_BLOCKS_FOR_SQUEEZENET1_0)
+                           ))
+
+def test_building_block(model_creator, input_sizes, ref_building_blocks):
+    model = model_creator()
+    nncf_config = get_empty_config(input_sample_sizes=input_sizes)
+    _, compressed_model = create_compressed_model(model, nncf_config)
+
+    building_blocks = get_building_blocks(compressed_model)
+    assert building_blocks == ref_building_blocks
```

Comment:
How is it that the ref object is a `List[List[str, str]]`, and the test object is a `List[BuildingBlock]` and the test passes?

Reply:
Fixed. Thanks.